src/peergos/shared/NetworkAccess.java

```diff
@@ -340,6 +340,16 @@ private static NetworkAccess build(ContentAddressedStorage dht,
                 });
     }
 
+    public CompletableFuture<Optional<?>> getFile(SharedItem item) {
```

Comment:
no need for this method, just use the one we were using

Reply:
But i need the item in the Pair. I'm sure it culd be done another way, but i need sleep.